packages/cubejs-backend-shared/src/logger.ts

```diff
@@ -0,0 +1,132 @@
+import SqlString from 'sqlstring';
+import R from 'ramda';
+
+interface logParams {
```

Comment:
Let's prefer PascalCase for interface names and keep each element on a separate line